tests/checkup_test.go

```diff
@@ -77,7 +77,7 @@ var _ = Describe("Checkup execution", func() {
 	})
 
 	It("should complete successfully", func() {
-		Eventually(getJobConditions, 5*time.Minute, 5*time.Second).Should(
+		Eventually(getJobConditions, 15*time.Minute, 5*time.Second).Should(
```

Comment:
since this checkup is supposed to run for long periods of time, this timeout is bound to collide with the oslat testDuration and fail the test prematurely. Can you do something in the terms of [this](https://github.com/kiagnose/kubevirt-dpdk-checkup/blob/596efb53f4227c2c73348aeb853e3080c8d2b0d9/tests/checkup_test.go#L98)?

Reply:
Sure, I'll do it in a follow-up if its ok with you.

Reply:
np